mysql-test/suite/innodb/t/percona_changed_page_bmp_1368530.test

```diff
@@ -0,0 +1,16 @@
+# Test that there is no conflict between the I_S plugin and the changed page bitmap option names (LP:1105726)
+--source include/have_innodb.inc
+--source include/have_debug.inc
+
+# If the bug is not present, server starts
+DROP DATABASE test;CREATE DATABASE test;
+USE test;
+SET @@GLOBAL.innodb_track_changed_pages=FALSE;
```

Comment:
This appears to test something else than what needed fixing. Here we start if var=FALSE, set it to FALSE again (succeed), then TRUE (succeed but should have failed instead), then FALSE again (fail).